back/app/models/project.rb

```diff
@@ -104,6 +104,10 @@ class Project < ApplicationRecord
     against: %i[title_multiloc description_multiloc description_preview_multiloc slug],
     using: { tsearch: { prefix: true } }
 
+  pg_search_scope :search_by_title,
+    against: :title_multiloc,
+    using: { tsearch: { prefix: true } }
```

Comment:
Probably need an index on `title_multiloc` or this may be slow on larger platforms. Although the number of projects to search is never going to be massive in the same way as ideas might be.

Reply:
Yeah probably fine in retrospect. Can always be added later if things look slow